hw_19042018/main.rb

```diff
@@ -0,0 +1,56 @@
+load 'customer.rb'
+require ('mysql2') 
+$client = Mysql2::Client.new(:host => "localhost", :username => "root", :password => "Abc123@@")
+$client.query("use hw19042");
```

Comment:
Use single quote for normal string.
Use double quote if insert variable to string.